registration-processor/core-processor/registration-processor-biometric-extraction-stage/src/main/java/io/mosip/registration/processor/stages/biometric/extraction/stage/BiometricExtractionStage.java

```diff
@@ -385,7 +382,7 @@ private IdResponseDTO addBiometricExtractiontoIdRepository(ExtractorDto dto,
 	 * @throws JsonParseException 
 	 * @throws RegistrationProcessorCheckedException 
 	 */
-	private ExtractorsDto getExtractors(String id) throws JSONException, ApisResourceAccessException, JsonParseException, JsonMappingException, JsonProcessingException, IOException, RegistrationProcessorCheckedException {
+	private ExtractorsDto getExtractors(String id) throws JSONException, ApisResourceAccessException, JsonParseException, JsonMappingException, JsonProcessingException, IOException, RegistrationProcessorCheckedException, IdrepoDraftReprocessableException, IdrepoDraftException {

```

Comment:
Remove the IDrepo exceptions in this place,it is calling PMS apis here

Reply:
dene.

Reply:
its not fixed,please fix it